Library/Homebrew/test/rubocops/cask/array_alphabetization_spec.rb

```diff
@@ -56,4 +56,17 @@
       end
     CASK
   end
+
+  it "ignores zap trash paths that have interpolation" do
+    expect_no_offenses(<<~CASK)
+      cask "foo" do
+        url "https://example.com/foo.zip"
+
+        zap trash: [
+          "~/Library/Application Support/Foo",
+          "~/Library/Application Support/Bar\#{version.major}",
+        ]
+      end
+    CASK
+  end
```

Comment:
@issyl0 I'm not seeing an issue with interpolation in paths locally, unless there's a specific way to trigger it not working.